synapse/python_dependencies.py

```diff
@@ -72,6 +72,7 @@
     # Twisted 18.7.0 requires attrs>=17.4.0
     "attrs>=17.4.0",
     "netaddr>=0.7.18",
+    "jaeger-client>=4.0.0",
```

Comment:
We probably want to make this an optional dependency so that people only need to install it if they want it. (This is mainly for downstream packagers who probably don't have `jaeger-client` in their repos)